### DISCUSSION TITLE
Help with noname IR module based on TYWE3S

### DISCUSSION THREAD

**Discussion by dansity** [arendst/Tasmota#22332] [Q&A] [answered]
Hello,
I have this IR module for a long time and I could use it now. It was branded V-TAC and it no longer has support. The app was something similar to ewelink but it is not pairing with ewelink. I cannot use it in its original form, I have flashed Tasmota on it. Flashing was straightforward with Tasmotizer.
As I can see (and tested continuity with multimeter) the GPIO pins, GPIO0 is IRsend and 5 is receive. All other pins are disconnected.
I have flashed tasmota-ir version and trying to make it work and recognize any IR payload from a TV remote but I get nothing in the console.
Any help appreciated where to start solving this.
![image](https://github.com/user-attachments/assets/2a833ec9-6b76-4b6f-96cc-5a11c6cafe25)

![IMG_20241020_154927971](https://github.com/user-attachments/assets/8f387041-c40a-4794-84e0-9cc0ce19749d)
![IMG_20241020_154920823](https://github.com/user-attachments/assets/bd81d789-0f38-46cd-a739-52e34faa0e31)

**Comment by sfromis** [comment]
You should be able verify which gpio is the receive pin is by configuring it as a Counter. Then anything detected would result in the counter value increasing as you push buttons on the remote.

**Reply by dansity** [reply]
thank you for your reply.
I have figured out that this is an YTF IR blaster but with a different brand name. So it has a template in Tasmota. The template says it is indeed GPIO5 for IRrecv same as I measured with the multimeter. I have increased the sensitivity with SetOption38 (6..255) to 200 and still I get no input in the console. For the record I'm using a generic replacement TV remote, so this is not some fancy HVAC remote.
The IRsend is working, I put a random payload in the console and I can see my IR leds blinking with my phone camera.
The IR receiver is a mystery still.

**Reply by sfromis** [reply]
Did you try my suggestion, to see if *anything* is picked up?

**Reply by dansity** [reply]
I dont have Counter as a selectable option for any GPIO. Is that bundled in tasmora-ir version?

**Reply by sfromis** [reply, marked as answer]
Sounds strange. With both current `tasmota.bin.gz`, and also `tasmota-ir.bin.gz`, I'm seeing `Counter` available in each gpio dropdown, for both Configure Template and Configure Module, same place as where I can select `IRrecv` and `IRsend`.
![image](https://github.com/user-attachments/assets/efd5350a-c27e-440a-897a-4944728658be)

IDK why you've flashed a very outdated Tasmota version 9.3.1, but from what I see, the same should be available there.

**Reply by dansity** [reply]
I dont know why I managed to download an old version. I checked now in google and for any keyword its only the latest repos, I have no idea how even I was managed to find an old one.
Anyway.. thank you for pointing this out. This was the issue. I updated it and now it is working as it should be on GPIO5.

Thanks for your help
